cypress/e2e/tests/custom-metrics/issues_tracker.test.ts

```diff
@@ -0,0 +1,109 @@
+import { createMultipleApplications, login } from "../../../utils/utils";
+import { CredentialType, JiraType, SEC } from "../../types/constants";
+import * as data from "../../../utils/data_utils";
+import { MigrationWave } from "../../models/migration/migration-waves/migration-wave";
+import { Assessment } from "../../models/migration/applicationinventory/assessment";
+import { Jira } from "../../models/administration/jira-connection/jira";
+import { JiraIssue } from "../../models/administration/jira-connection/jira-api.interface";
+import { JiraCredentials } from "../../models/administration/credentials/JiraCredentials";
+import { Metrics } from "../../models/migration/custom-metrics/custom-metrics";
+
+const now = new Date();
+now.setDate(now.getDate() + 1);
+
+const end = new Date(now.getTime());
+end.setFullYear(end.getFullYear() + 1);
+
+let applications: Assessment[];
+let migrationWave: MigrationWave;
+let jiraInstance: Jira;
+let jiraCredentials: JiraCredentials;
+const metrics = new Metrics();
+const metricName = "konveyor_issues_exported_total";
+let counter: number;
+
+// Automates Polarion TC 337
+describe(["@tier2"], "Custom Metrics - Count the total number of issues exported", function () {
+    before("Create test data", function () {
+        login();
+        applications = createMultipleApplications(2);
+
+        migrationWave = new MigrationWave(
+            data.getRandomWord(8),
+            now,
+            end,
+            null,
+            null,
+            applications
+        );
+        migrationWave.create();
+
+        jiraCredentials = new JiraCredentials(
+            data.getRandomCredentialsData(CredentialType.jiraBasic, null, true)
+        );
+        jiraCredentials.create();
+
+        jiraInstance = new Jira({
+            name: data.getRandomWord(5),
+            url: Cypress.env("jira_atlassian_cloud_url"),
+            credential: jiraCredentials,
+            type: JiraType.cloud,
+        });
+        jiraInstance.create();
+
+        metrics.getValue(metricName).then((counterValue) => {
+            counter = counterValue;
+        });
+    });
+
+    it("Export issues to Jira and validate the metrics counter", function () {
+        let projectName = "";
+
+        jiraInstance
+            .getProject()
+            .then((project) => {
+                expect(!!project).to.eq(true);
```

Comment:
You do not have assert anything for Jira fields in this test . 
These are covered in waves test . 
Just export issues to JIRA .

Correct @abrugaro ? This step can be avoided ?

Reply:
Yes, it worked and removed the assertion for jira insatance. Just kept for migrationWave. Thanks!

Reply:
That's right!